src/Routes/Routes.tsx

```diff
@@ -113,18 +105,14 @@ export const Routes = memo(() => {
         const MainComponent = route.main
 
         return (
-          <PrivateRoute
-            key={isUnstableRoute ? Date.now() : 'privateRoute'}
-            path={route.path}
-            hasWallet={hasWallet}
-          >
+          <PrivateRoute key={'privateRoute'} path={route.path} hasWallet={hasWallet}>
```

Comment:
q: does having multiple of the same key cause react warning spew?

Reply:
Surprisingly not in the context of react-router `<Route />`s, no! 
<img width="597" alt="image" src="https://github.com/user-attachments/assets/7bf0554c-fbd2-4054-a3eb-2602106fdb74">

For historical context, back in the early days of the app we had a *lot*, and I mean a lot of bugs and perf issues related to missing keys in `<Route />`s so the same was added for all, iirc we couldn't use a unique one for each since that would cause re-renders. I could absolutely be wrong here but a case of "probably better not to touch this" given the very big implications!

Reply:
I'm more surprised by the fact that every occurences of `<PrivateRoute />` includes all the public routes

```jsx
    <Switch location={location}>
      <Route {...rest} path='/markets' />
      <Route {...rest} path='/assets' />
      <Route {...rest} path='/rfox' />
      <Route {...rest} path='/explore' />
      <Route {...rest} path='/pools' />
      <Route {...rest} path='/earn' />
      <Route {...rest} path='/buy-crypto' />
      <Route {...rest} path='/assets' />
      <Route {...rest} path='/flags' />
      <Route {...rest} path='/fox' />
      {hasWallet ? <Route {...rest} /> : <Redirect to={to} />}
    </Switch>
    ```

Reply:
soon rambo'd in theaters near you